src/services/user/useOwner.ts

```diff
@@ -0,0 +1,74 @@
+import { useQuery } from '@tanstack/react-query'
+import { useParams } from 'react-router-dom'
+import { z } from 'zod'
+
+import Api from 'shared/api'
+import { rejectNetworkError } from 'shared/api/helpers'
+
+const OwnerSchema = z.object({
+  ownerid: z.number().nullish(),
+  username: z.string().nullish(),
+  avatarUrl: z.string().nullish(),
+  isCurrentUserPartOfOrg: z.boolean().nullish(),
+  isAdmin: z.boolean().nullish(),
+})
+
+export type Owner = z.infer<typeof OwnerSchema>
+
+const RequestSchema = z.object({
+  owner: OwnerSchema.nullish(),
+})
+
+interface URLParams {
+  provider: string
```

Comment:
nit: This can now be typed out as Provider from `import { Provider } from 'shared/api/helpers'`

Reply:
nice! fixed!